### DISCUSSION TITLE
v2.8.66

### DISCUSSION THREAD

**Discussion by fiftin** [semaphoreui/semaphore#1039] [Announcements]
* Slack alerts added.
* Added support of local projects without Git.
* User-friendly selecting of repository type:

![image](https://user-images.githubusercontent.com/914224/193407585-25a8bad5-c33a-45e8-ad68-96ef7fc6fe39.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible-semaphore/semaphore/releases/tag/v2.8.66'>v2.8.66</a>.</em>

**Comment by leandroscardua** [comment]
Thank you
I really appreciate the support to freebsd on this release.

👯

**Comment by fiftin** [comment]
Hi @leandroscardua 
Have you tested v2.8.66 on FreeBSD? I have no ability to do this.

**Comment by leandroscardua** [comment]
HI @fiftin ,

Yes, I`m able to install on FreeBSD using the version v2.8.66, please take a look on it uses jail.

https://github.com/leandroscardua/BastilleBSD-Templates-Semaphore/blob/main/Bastillefile

FreeBSD version: 13.1 (with latest updates)
mySQL: 8.0

**Comment by leandroscardua** [comment]
Also, with the latest v2.8.67 👯

**Comment by tovoro** [comment]
I just upgraded but I dont really understand. Is the Docker Version more up to date than the package release? Can somebody elaborate?

**Reply by fiftin** [reply]
No, latest Docker version = latest package version.

https://hub.docker.com/r/semaphoreui/semaphore/tags

**Comment by leandroscardua** [comment]
@fiftin ,

Please, let me know, if you need help to test the FreeBSD package.
I just open a PR to add the ansible-semaphore as a template on [bastilleBSD](https://bastillebsd.org/)
